app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt

```diff
@@ -58,16 +59,17 @@ fun bindImage(imgView: ImageView, imgUrl: String?) {
 @BindingAdapter("marsApiStatus")
 fun bindStatus(statusImageView: ImageView, status: MarsApiStatus?) {
     when (status) {
-        MarsApiStatus.LOADING -> {
+        LOADING -> {
             statusImageView.visibility = View.VISIBLE
             statusImageView.setImageResource(R.drawable.loading_animation)
         }
-        MarsApiStatus.ERROR -> {
+        ERROR -> {
             statusImageView.visibility = View.VISIBLE
             statusImageView.setImageResource(R.drawable.ic_connection_error)
         }
-        MarsApiStatus.DONE -> {
+        DONE -> {
             statusImageView.visibility = View.GONE
         }
```

Comment:
Please revert this change, why do we need this anyway? This will break the code

Reply:
Reverted.